addons/talo/utils/crypto_manager.gd

```diff
@@ -1,11 +1,11 @@
-class_name TaloCryptoManager extends Node
+class_name TaloCryptoManager extends RefCounted
 
-var _key_file_path = "user://ti.bin"
+const _KEY_FILE_PATH := "user://ti.bin"
```

Comment:
👌 this definitely should be a const, good spot. would be great if this (along with the other const changes) could be in a separate PR.